test/service/test_response.py

```diff
@@ -1230,10 +1224,10 @@ def test_filter_with_none(self):
                 response.raise_for_status()
                 response_json = response.json()
                 organism_age_units = {
-                    oau
+                    oa if oa is None else oa['unit']
```

Comment:
```suggestion
                    None if oa is None else oa['unit']
```

to avoid the mental gymnastics